charts/data-dashboard-backend/README.md

```diff
@@ -44,7 +43,7 @@ API for data in the data dashboard
 | ingress.enabled | bool | `true` | Enable ingress controller resource |
 | ingress.ingressClassName | string | `"nginx"` | Ingress class name |
 | ingress.annotations | object | check values.yaml | Annotations that define default ingress class, certificate issuer |
-| ingress.path | string | `"/api"` | Path within the url structure |
+| ingress.path | string | `"/dashboard-data"` | Path within the url structure |
```

Comment:
Is there a need for documentation to be updated for this? Or maybe the `radar-home` chart?

Reply:
@keyvaann No I do not think so. Since this is only a backend (used by aRMT for now) we not need it in radar-home. The change in this PR is there so we no longer have to set this in helmfile (done for MySkin).